spec/builtin/dns/srv_enum_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/dns/srv_enum'
+
+describe Ronin::Recon::DNS::SRVEnum do
+  describe "#process" do
+    context "when there are hosts in the domain" do
+      let(:domain) { Ronin::Recon::Values::Domain.new("gmail.com") }
+      let(:hosts) do
+        %w[
+          imap.gmail.com
+          smtp.gmail.com
+          pop.gmail.com
+          calendar.google.com
+          calendar.google.com
+        ]
+      end
+
+      it "must yield Host values" do
+        yielded_values = []
+
+        subject.process(domain) do |value|
+          yielded_values << value
+        end
+
+        expect(yielded_values).to_not be_empty
+        expect(yielded_values).to all(be_kind_of(Ronin::Recon::Values::Host))
+        expect(yielded_values.map(&:name).reject(&:empty?)).to eq(hosts)
```

Comment:
If `SRVEnum#process` is yielding empty values, that seems like a bug.  Those should be filtered out in `SRVEnum#process`.

Reply:
Should be fixed in 979e171c3a3512ae18004eb1c043765dc1c4907e.